cmd/lifecycle/restorer.go

```diff
@@ -116,7 +116,7 @@ func (r *restoreCmd) Exec() error {
 			}
 		} else if r.supportsTargetData() && needsUpdating(analyzedMD.RunImage) {
 			cmd.DefaultLogger.Debugf("Updating run image info in analyzed metadata...")
-			remoteRunImage, err = remote.NewImage(runImageName, r.keychain)
+			remoteRunImage, err = remote.NewImage(runImageName, r.keychain, remote.FromBaseImage(runImageName))
```

Comment:
I can't figure out why this would work -- maybe it has the effect of (re-)loading the image?

Reply:
`remote.NewImage` in `imgutil` sadly doesn't work the same as `remote.Image` in GGCR - without the base image option, all the getters on the image return data for an empty image! I sadly forgot this fact when writing this code.